HHS/OPRE-OPS
# feat: ✨ adds CanComboBox

## What changed

- adds CanComboBox based on `react-select`

## Issue

#2049 #2169 

## How to test

1. Go to Agreements SC& BLI tab
2. create a BLI with a CAN
3. test type ahead
4. after creating/editing a BLI, the form should clear
5. 💰 profit

## Screenshots

_If relevant, e.g. for a front-end feature_

## Definition of Done Checklist
